src/PASopa/Program.cs

```diff
@@ -17,6 +17,8 @@ static void Main(string[] args)
         {
             Console.WriteLine($"MsApp/Source converter. Version: {SourceSerializer.CurrentSourceVersion}");
 
+            var warningText = "Warning: {0} is in preview, and functionality is not guaranteed. Use extreme caution if using in a production environment. For more information see aka.ms/paccanvas";
```

Comment:
nit: I'd declare this as a static constant outside of `Program`. Just makes it a little easier to find for anyone changing in the future.